### DISCUSSION TITLE
en/

### DISCUSSION THREAD

**Discussion by giscus** [typed-sigterm/vitepress-giscus#3] [Comments]
# en/

Out-of-the-box Giscus support for VitePress.

https://vitepress-giscus.by-ts.top/en/

**Comment by typed-sigterm** [comment]
Hello! :tada: Demo area here. In actual projects, you can place this anywhere.

**Test markdown features...**

# H1
followed by some text

## H2
followed by some text

### H3
followed by some text

#### H4
followed by some text

##### H5
followed by some text

###### H6
followed by some text

Auto-detected link: https://giscus.vercel.app

Can render `inline code`, *italics*, **strong**, :smile:

> here is blockquote
> test `inline code` *italics* **strong** :smile:

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id sem purus, eu commodo tortor. Donec malesuada ultricies dolor a eleifend. In hac habitasse platea dictumst. Vivamus a faucibus ligula. Nullam molestie tristique arcu, eu elementum metus ultricies sed. Aenean luctus congue lectus, vitae semper erat rhoncus non. Nulla facilisi.

Horizontal rule:

------

|Table Header 1|Table Header 2           |
|--------------|-------------------------|
|Content       | https://github.com/laymonage/giscus       |
|Content       |http://github.com:\<test\>|
I'm below the table.

A table with an empty cell and unaligned indenting.

|Table Header 1|Table Header 2|
|--------------|--------------|
|Content  |  Content        |
|Content       |      |

    def this_is
      puts "some #{4-space-indent} code"
    end

<code>
def this_is
  puts "some #{code tag} code"
end
</code>

<pre>
def this_is
  puts "some #{pre tag} code"
end
</pre>

```
def this_is
  puts "some #{fenced} code"
end
```

```ruby
class Classy
  def this_is
    puts "some #{colored} ruby code with ruby syntax highlighting"
    @someobj.do_it(1, 2)
  end
end
```

```javascript
var test = function this_is(){
  console.log("some" + colored + "javascript code with javascript syntax highlighting really long");
}
```

```clojure
(defproject myproject "0.5.0-SNAPSHOT"
  :description "Some clojure code with syntax highlighting."
  :dependencies [[org.clojure/clojure "1.5.1"]]
  :plugins [[lein-tar "3.2.0"]])
```

```js
var test = function this_is(){
  console.log("language declared as 'js' instead");
}
```

```bogus_language
var test = function this_is(){
  console.log("language declared as bogus_language");
}
```

```diff
- print('hello world')
+ print('Hello, world!')
```

https://github.com/typed-sigterm/vitepress-giscus/blob/8f5dffb30182efaa7ed1c5e94df8ea41dedb33f6/src/index.ts#L1-L2

![image link](https://octodex.github.com/images/hula_loop_octodex03.gif)

Test from https://github.com/suan/github-flavored-markdown-test

Test math renderer error:

${{{¶∆✓¥€¢|π✓©&#//)()(%£€{π√×¶obrigada€¢¥™£®}\\∆×]✓\\€¢£¥✓\\pelas#@\")(\"\"¢¥¥π÷✓]¶×∆¢π°flores$@&€¢£¥amei////)($#®€¢£¥}💖

When  $a \ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$ and they are 
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}$$

**The Cauchy-Schwarz Inequality**

$$\left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right)$$
